test/components/plots/plotTests.ts

```diff
@@ -217,13 +217,13 @@ describe("Plots", () => {
       var renderArea1 = svg.append("g");
       var renderArea1Selection = renderArea1.append("circle").attr("cx", 100).attr("cy", 100).attr("r", 10);
       (<any> mockDrawer1).setup = () => (<any> mockDrawer1)._renderArea = renderArea1;
```

Comment:
You may decide to remove the `setup`, but there is no need for the `<any>` cast here. Feel free to ignore this comment as it is not part of the PR